ENV.MD

```diff
@@ -0,0 +1,427 @@
+
+##### SA_KILL
+
+  - **6.31** => triggers spam evasive actions
+
+Note: this spamassassin setting needs `ENABLE_SPAMASSASSIN=1`. By default, the mailserver is configured to quarantine spam emails. If emails are quarantined, they are compressed and stored in a location dependent on the ONE_DIR setting above. If `ONE_DIR=1` the location is /var/mail-state/lib-amavis/virusmails/. If `ONE_DIR=0` it is /var/lib/amavis/virusmails/. These paths are inside the docker container. To inhibit this behaviour and deliver spam emails, set this to a very high value e.g. 100.0.
+
+##### SA_SPAM_SUBJECT
+
+  - **\*\*\*SPAM\*\*\*** => add tag to subject if spam detected
+
+Note: this spamassassin setting needs `ENABLE_SPAMASSASSIN=1`
+
+## Fetchmail
+
+##### ENABLE_FETCHMAIL
+  - **0** => `fetchmail` disabled
+  - 1 => `fetchmail` enabled
+
+##### FETCHMAIL_POLL
+  - **300** => `fetchmail` The number of seconds for the interval
+
+## LDAP
+
+##### ENABLE_LDAP
+
+  - **empty** => LDAP authentification is disabled
+  - 1 => LDAP authentification is enabled
+  - NOTE:
+    - A second container for the ldap service is necessary (e.g. [docker-openldap](https://github.com/osixia/docker-openldap))
+    - For preparing the ldap server to use in combination with this container [this](http://acidx.net/wordpress/2014/06/installing-a-mailserver-with-postfix-dovecot-sasl-ldap-roundcube/) article may be helpful
+
+##### LDAP_START_TLS
+
+  - **empty** => no
+  - yes => LDAP over TLS enabled for Postfix
+
+##### LDAP_SERVER_HOST
+
+  - **empty** => mail.domain.com
+  - => Specify the dns-name/ip-address where the ldap-server
+  - NOTE: If you going to use the mailserver in combination with docker-compose you can set the service name here
+
+##### LDAP_SEARCH_BASE
+
+  - **empty** => ou=people,dc=domain,dc=com
+  - => e.g. LDAP_SEARCH_BASE=dc=mydomain,dc=local
+
+##### LDAP_BIND_DN
+
+  - **empty** => cn=admin,dc=domain,dc=com
+  - => take a look at examples of SASL_LDAP_BIND_DN
+
+##### LDAP_BIND_PW
+
+  - **empty** => admin
+  - => Specify the password to bind against ldap
+
+##### LDAP_QUERY_FILTER_USER
+
+  - e.g. `(&(mail=%s)(mailEnabled=TRUE))`
+  - => Specify how ldap should be asked for users
+
+##### LDAP_QUERY_FILTER_GROUP
+
+  - e.g. `(&(mailGroupMember=%s)(mailEnabled=TRUE))`
+  - => Specify how ldap should be asked for groups
+
+##### LDAP_QUERY_FILTER_ALIAS
+
+  - e.g. `(&(mailAlias=%s)(mailEnabled=TRUE))`
+  - => Specify how ldap should be asked for aliases
+
+##### LDAP_QUERY_FILTER_DOMAIN
+
+- e.g. `(&(|(mail=*@%s)(mailalias=*@%s)(mailGroupMember=*@%s))(mailEnabled=TRUE))`
+- => Specify how ldap should be asked for domains
+
+##### DOVECOT_TLS
+
+  - **empty** => no
+  - yes => LDAP over TLS enabled for Dovecot
+
+## Dovecot
+
+The following variables overwrite the default values for ```/etc/dovecot/dovecot-ldap.conf.ext```.
+
+##### DOVECOT_USER_FILTER
+
+  - e.g. `(&(objectClass=PostfixBookMailAccount)(uniqueIdentifier=%n))`
```

Comment:
this could also mention the default values. maybe its sufficient to say that unless set dovecot-ldap.conf.ext will not be touched